docs/amm.md

```diff
@@ -0,0 +1,100 @@
+# CoW AMM
+
+CoW AMM is an automated market maker running on top of CoW Protocol.
+
+## How it works
+
+The AMM contract itself is a dedicated Safe multisig.
+It stores reserves of two tokens and allows anyone to create orders between these two tokens on CoW Protocol as long as the trade doesn't decrease the product of the reserves stored on the contract.
+
+The order is based on the composable CoW framework: this repository only specified the code needed for the dedicated handler and the related price oracles.
```

Comment:
```suggestion
The order is based on the `ComposableCoW` framework: this repository only specifies the code needed for the dedicated handler and the related price oracles.
```

For documentation conventions, if we are referring to a piece of code, we use fixed-width, otherwise we use the marketing term, ie. programmatic order framework).